descriptors/ModuleDescriptor-template.json

```diff
@@ -2,6 +2,68 @@
   "id": "${artifactId}-${version}",
   "name": "Invoice CRUD module",
   "provides": [
+    {
+      "id": "invoice-storage.invoice",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/invoice-storage/invoices",
+          "permissionsRequired": ["invoice.collection.get"]
```

Comment:
Please use `invoice-storage.` prefix (as you already defined in `permissionSets`)

Reply:
Corrected